### PR TITLE
add qx util

### PR DESCRIPTION
**Pull request by luoshan654**
add qx  util
Add a progress bar when updating the block, and fix the bug of updating the block repeatedly.

### CODE REVIEW

File: console/command.go [hunk 1]
```diff
@@ -99,7 +99,7 @@ func LoadConfig(cmd *cobra.Command, args []string)  {
 	if cmd.Flag("pubwalletpass").Changed {
 		fileCfg.WalletPass = preCfg.WalletPass
 	}
-	log.SetLevel(log.TraceLevel)
+	//log.SetLevel(log.TraceLevel)
```
Comment (dindinw) [inline console/command.go]: the log level should be configurable instead of being comment off

File: console/console.go [hunk 2]
```diff
@@ -1,8 +1,13 @@
 package console
 
 import (
+	"encoding/hex"
 	"encoding/json"
 	"fmt"
+	"github.com/Qitmeer/qitmeer-lib/crypto/bip39"
```
Comment (dindinw) [inline console/console.go]: should not use the `qitmeer-lib` anymore, which obsoleted by `qitmeer/qx`

File: rpc/walletrpc/methods_test.go [hunk 3]
```diff
@@ -285,11 +284,20 @@ func test_wif(w *wallet.Wallet) error {
...
-	w, err := open_wallet()
-	if err != nil {
-		fmt.Println("open_wallet err:", err)
+	serializedPubKey, err := hex.DecodeString("049ed083b4d2a923c9736c3186444610de68576e4314b03e2b638b4738a1741c460f23f8921dbdf6cbf0a3b67fc53734ad5c05116f55d41abc8d78f10aa1e4f2a2")
+	pu,err:=secp256k1.ParsePubKey(serializedPubKey)
+	if err!=nil{
+		fmt.Print(err.Error())
 		return
 	}
+	fmt.Printf("pu.SerializeCompressed()：%x\n",pu.SerializeCompressed())
```
Comment (dindinw) [inline rpc/walletrpc/methods_test.go]: test method should not depend on seeing the stdout, you can use //output flag, see https://golang.org/pkg/testing/#hdr-Examples

File: rpc/walletrpc/methods_test.go [hunk 4]
```diff
@@ -285,11 +284,20 @@ func test_wif(w *wallet.Wallet) error {
...
+	fmt.Printf("pu.SerializeUncompressed()：%x\n",pu.SerializeUncompressed())
+
+	//w, err := open_wallet()
```
Comment (dindinw) [inline rpc/walletrpc/methods_test.go]: please don't use comment off

File: wallet/wallet.go [hunk 5]
```diff
@@ -95,24 +97,16 @@ func (wt *Wallet) Start() {
...
 
-		updateBlockTicker := time.NewTicker(60 * time.Second)
-		for {
-			select {
-			case <-updateBlockTicker.C:
-				log.Trace("Updateblock start")
-				wt.Updateblock(0)
-			}
-		}
+		//updateBlockTicker := time.NewTicker(60 * time.Second)
```
Comment (dindinw) [inline wallet/wallet.go]: I doubt the logic to remove the channel

File: console/command.go (hunk 1 above)
Comment (dindinw) [reply console/command.go]: And, we need to unify how to do the output, please use `log` instead of `fmt.println`

File: wallet/wallet.go (hunk 5 above)
Comment (luoshan654) [reply wallet/wallet.go]: The web mode and console mode are distinguished here, and they are processed in the updateblock method

**Pull request merged**